### DISCUSSION TITLE
Notepad++ Custom Language and Hints

### DISCUSSION THREAD

**Discussion by KnightCaptainU7** [exult/exult#594] [Show and tell]
In order to speed the writing of Usecode, I created a custom Notepad++ language (syntax highlighter) and a hints file (autocomplete). This makes writing new code easier for me, and I hope it will help others.

![image](https://github.com/user-attachments/assets/ecc3ef70-dd66-4e74-befb-0e65f60b7fac)

![image](https://github.com/user-attachments/assets/22f1452b-26b5-4232-b14d-5c04b928584a)

The hints stay up even as you add parameters.
![image](https://github.com/user-attachments/assets/78670921-77dc-45e7-90c0-8c3c538bc8ca)

https://github.com/KnightCaptainU7/UsecodeC-for-NotepadPP

**Comment by DominusExult** [comment]
pretty cool and useful!
